src/pkg/reg/adapter/harbor/v2/adapter.go

```diff
@@ -129,3 +129,14 @@ func (a *adapter) listArtifacts(repository string, filters []*model.Filter) ([]*
 	}
 	return filter.DoFilterArtifacts(artifacts, filters)
 }
+
+func (a *adapter) CanBeMount(digest string) (bool, string, error) {
+	repository, err := a.client.getRepositoryByBlobDigest(digest)
```

Comment:
Do we need to process the 404 here for the previous version of harbor?

Reply:
updated